frontend/taipy-gui/src/components/Taipy/DateRange.tsx

```diff
@@ -141,8 +154,9 @@ const DateRange = (props: DateRangeProps) => {
                                     slotProps={textFieldProps}
                                     label={props.labelStart}
                                     format={props.format}
+                                    sx={dateSx}
                                 />
-                                -
+                                <Typography>{separator}</Typography>
```

Comment:
Please tell me why this - I trust it makes sense, just asking for my culture.

Reply:
we need typography so that the stack can differentiate the blocks
separator because we might want to make it user changeable one day :-)